core/src/main/scala/com/mesosphere/usi/core/SchedulerFactory.scala

```diff
@@ -78,15 +87,14 @@ class SchedulerFactory private (
   }
 
   override def newSchedulerLogicGraph(snapshot: StateSnapshot): SchedulerLogicGraph = {
-    new SchedulerLogicGraph(client.calls, client.masterInfo.getDomain, snapshot, metrics)
+    new SchedulerLogicGraph(new MesosCalls(), client.masterInfo.getDomain, snapshot, metrics)
```

Comment:
Eg. here, we expect the master information. I have to admit that I do not like the current design. USI makes too many assumption.